src/main/views/features/unincorporated/business-address-manual-entry/business-address-manual-entry.njk

```diff
@@ -0,0 +1,111 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% extends "layouts/default.njk" %}
+{% set errors = pageProperties.errors %}
+{% block main_content %}
+  <form action="" method="POST">
+    {% include "partials/business_name.njk" %}
```

Comment:
Should there be conditional clause here for displaying the business name?

Reply:
No